AirCasting/SessionViews/SessionCardView.swift

```diff
@@ -117,11 +117,15 @@ struct SessionCardView: View {
     }
 
     private func selectDefaultStreamIfNeeded(streams: [MeasurementStreamEntity]) {
-        if selectedStream == nil {
-            if let newStream = session.streamWith(sensorName: session.userInterface?.sensorName ?? "") {
-                return selectedStream = newStream
+        // AirBeam is giving us two streams at first (F, RH) when doing mobile active, which prevents us from switching to PM2.5 as selected stream.
+        // Solution below is not ideal, as in the future (this was written in 29/08/2022) the number of streams can change. Yet, it is working well - selecting PM2.5 as deafult stream for every session.
+        if streams.count == 5 || streams.count == 1 {
+            if selectedStream == nil {
+                if let newStream = session.streamWith(sensorName: session.userInterface?.sensorName ?? "") {
+                    return selectedStream = newStream
```

Comment:
this is fancy, but let's not do it 😅 and change the order to a standard one: `selectedStream = newStream; return`

Reply:
> When I did it without breakpoints it did work, but this means this solution is not "bulletproof"

Where did you put those breakpoints at first? I was trying to reproduce, but even with using those breakpoints, I can get it right (PM2.5 is selected). If something like this happens (without breakpoints it works only), can we be sure it can be made "bulletproof" somehow?- I mean, we are using BT and receiving data constantly so maybe this is something we are not able to handle. I am asking because I don't have another idea for this right now.